msal_extensions/persistence.py

```diff
@@ -87,11 +106,32 @@ def save(self, content):
     def load(self):
         # type: () -> str
         """Load content from this persistence"""
-        with open(self._location, 'r') as handle:
-            return handle.read()
+        try:
+            with open(self._location, 'r') as handle:
+                return handle.read()
+        except EnvironmentError as exp:  # EnvironmentError in Py 2.7 works across platform
+            if exp.errno == errno.ENOENT:
+                raise PersistenceNotFound(
+                    message=(
```

Comment:
I think we really shouldn't be encouraging the use of this persistence layer as a public API. Although we already have Azure CLI using it for one of their use cases, we don't really expect people to be using this load() and save() functions for storing msal cache. In such a case we should make sure this persistence not found error is never raised to anyone using msal extensions with the token_cache layer  only.

Reply:
We discussed this offline, that this persistence layer is mean to be part of the public API. So, no actionable item on this part, for now. We could, however, consider to even make this persistence layer a new library by itself. But that is a different topic that we can discuss offline.